repos/system_upgrade/common/libraries/dnfconfig.py

```diff
@@ -19,6 +19,8 @@ def get_leapp_packages():
     generic = ['leapp', 'snactor']
     if get_source_major_version() == '7':
         return generic + ['python2-leapp', 'leapp-upgrade-el7toel8']
+    if get_source_major_version() == '9':
+        return generic + ['python3-leapp', 'leapp-upgrade-el9toel10']
 
     return generic + ['python3-leapp', 'leapp-upgrade-el8toel9']
```

Comment:
the problem the root cause of the whole problem - that  the function is duplicating definition of related packages again. to be compatible with previous versions, let's keep the function (we can deprecated later) but use instead `leapp.libraries.common.rpms.get_leapp_packages` to obtain the list of the framework, repository, and tools packages for the source OS major version. The function is written in the way that does already the thing, so something like:

```python
from leapp.libraries.common import rpms

def get_leapp_packages():
    return rpms.get_leapp_packages()
```

should do the thing and we drop the re-definition

Reply:
hmmm....silly me - I should posptone review at 1am. we do not need to even deprecate it. we can just remove it and import the other one...

```
from leapp.libraries.common.rpms import get_leapp_packages
```

as the default behaviour of the other function reflects the original purpose of this one, it's backward compatible so we do not need to even deal with the deprecation...